webapi/src/test/scala/org/knora/webapi/e2e/v1/StandoffV1R2RSpec.scala

```diff
@@ -149,6 +149,19 @@ class StandoffV1R2RSpec extends R2RSpec {
 
         val pathToHTML = "_test_data/test_route/texts/HTML.xml"
 
+        val paramsCreateHTMLExtMappingFromXML =
```

Comment:
Maybe you wanted to add new files but in the end you just added more to the existing files?